tests/swap_test.go

```diff
@@ -295,3 +244,81 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 
 	})
 })
+
+func min(a, b int) int {
```

Comment:
@oshoval Thanks a lot for the improvements!

Do you think its possible to utilize `math.Min` here instead?

Reply:
Trying first this one
https://go-review.googlesource.com/c/go/+/498495/3/src/builtin/builtin.go

there is a new built - in since 1.21

Reply:
Used built-in, it compiled locally
ptal